test/unit/spec/v2/view-builder/views/RedirectIdvView_spec.js

```diff
@@ -30,11 +30,10 @@ describe('v2/view-builder/views/idp/RedirectIdvView', function() {
         return [];
       });
 
+      appStateTriggerSpy = jest.spyOn(appState, 'trigger');
+
       const currentViewState = {
-        name: 'challenge-authenticator',
-        relatesTo: {
-          value: currentAuthenticator,
-        },
+        name: 'redirect-idverify',
```

Comment:
Were the view state settings for the test incorrect before?

Reply:
Yes earlier one is wrong. we need the `name` for it to display the correct UI.